### PR TITLE
Add secrets to excluded apps by default

### LINKED ISSUE #63: Add Secrets password manager to default Excluded apps.
**Is your feature request related to a problem? Please describe.**
When you copy password or login from Secrets password manager it is added to database of pano.

**Describe the solution you'd like**
Secrets should be added to excluded apps so passwords aren't stored in database.

**Describe alternatives you've considered**
None

**Additional context**
Secrets is only available as flatpak on flathub so the name of the window is "org.gnome.World.Secrets"

### PR DESCRIPTION
**Pull request by MontyQI**
closes #63

### CODE REVIEW

File: resources/schemas/org.gnome.shell.extensions.pano.gschema.xml [hunk 1]
```diff
@@ -65,7 +65,7 @@
       </description>
     </key>
     <key name="exclusion-list" type="as">
-      <default>['Bitwarden', '1Password', 'KeePassXC']</default>
+      <default>['Bitwarden', '1Password', 'KeePassXC', 'org.gnome.World.Secrets']</default>
```
Comment (oae) [inline resources/schemas/org.gnome.shell.extensions.pano.gschema.xml:68]: Thes are `WM_CLASS` property of the window. You can check it with `xprop` command. It should be `secrets`.

image

**Review by oae — COMMENTED** [review]
Thank you for the pr. I have a comment on the change. Can you check it?

File: resources/schemas/org.gnome.shell.extensions.pano.gschema.xml (hunk 1 above)
Comment (MontyQI) [reply resources/schemas/org.gnome.shell.extensions.pano.gschema.xml:68]: When I set it to secrets it still captures what i copy
image
But when it is set to org.gnome.World.Secrets it doesn't
image

Comment (oae) [reply resources/schemas/org.gnome.shell.extensions.pano.gschema.xml:68]: Can you share your `xprop` output?

Comment (MontyQI) [reply resources/schemas/org.gnome.shell.extensions.pano.gschema.xml:68]: I think it has something to do with wayland. I'm using it so I can't use xprop but if i use lg it says that wm_class is org.gnome.World.Secrets.
image

Comment (oae) [reply resources/schemas/org.gnome.shell.extensions.pano.gschema.xml:68]: oh. That makes sense. I am on X11. I think we should add both `secrets` and `org.gnome.World.Secrets` then.

**Pull request merged**